doc/development/deprecations.rst

```diff
@@ -88,6 +83,11 @@ for details on how to port your legacy code to the new system. The following fun
 Completed deprecation cycles
 ----------------------------
 
+* The ``KerasLayer`` in ``qml.qnn.keras`` has been removed because Keras 2 is no longer actively maintained.  Please consider using a different machine learning framework.
```

Comment:
```suggestion
* The ``KerasLayer`` class in ``qml.qnn.keras`` has been removed because Keras 2 is no longer actively maintained.  Please consider using a different machine learning framework, like [Pytorch](https://pennylane.ai/qml/demos/tutorial_qnn_module_torch) or [JAX](https://pennylane.ai/qml/demos/tutorial_How_to_optimize_QML_model_using_JAX_and_Optax).
```

I think we can use sphinx directives for demos here, right? If so, I suggest we use those instead

Reply:
let's try